File: velolabs_repos/lattis_android/app/src/main/java/com/lattis/ellipse/data/network/model/response/GetCurrentUserStatusPayloadResponse.java
package com.lattis.ellipse.data.network.model.response;

import com.google.gson.annotations.SerializedName;

/**
 * Created by ssd3 on 4/12/17.
 */

public class GetCurrentUserStatusPayloadResponse {

    @SerializedName("trip")
    public GetCurrentUserStatusTripResponse getCurrentUserStatusTripResponse;

    @SerializedName("active_booking")
    public GetCurrentUserActiveBookingStatusResponse getCurrentUserActiveBookingStatusResponse;

    @SerializedName("support_phone")
    private String support_phone;

    @SerializedName("on_call_operator")
    private String on_call_operator;

    public String getSupport_phone() {
        return support_phone;
    }

    public String getOn_call_operator() {
        return on_call_operator;
    }

}

File: velolabs_repos/lattis_android_v2/data/src/main/java/com/lattis/data/entity/response/user/current_status/GetCurrentUserStatusPayloadResponse.kt
package com.lattis.data.entity.response.user.current_status

import com.google.gson.annotations.SerializedName
import com.lattis.domain.models.DockHub
import com.lattis.domain.models.Reservation
import com.lattis.domain.models.UserCurrentStatus

class GetCurrentUserStatusPayloadResponse {
    @SerializedName("trip")
    var getCurrentUserStatusTripResponse: GetCurrentUserStatusTripResponse? = null
    @SerializedName("active_booking")
    var getCurrentUserActiveBookingStatusResponse: GetCurrentUserActiveBookingStatusResponse? = null
    @SerializedName("support_phone")
    val support_phone: String? = null
    @SerializedName("on_call_operator")
    val on_call_operator: String? = null

    @SerializedName("reservation")
    val reservation:Reservation?=null

    @SerializedName("vehicle")
    val vehicle:UserCurrentStatus.Vehicle?=null

    @SerializedName("hub")
    val dockHub:DockHub?=null


}